garden_ai/garden.py

```diff
@@ -79,3 +81,65 @@ def _authenticate(self):
             on_refresh=self.auth_key_store.on_refresh,
         )
         return authorizer
+
+    def create_garden(self, **kwargs):
+        """Initialize and return a new Garden object
+
+        If invoked with keyword arguments corresponding to field names (e.g.
+        `title` or `creator`) are passed directly to the `Garden` model, making
+        it possible for the user to provide necessary info upfront to be parsed
+        and validated at initialization, or to incrementally provide field
+        values via simple assignment to attributes of the returned object
+        directly.
+
+        Parameters
+        ----------
+        **kwargs :
+            Metadata for the new Garden object. Keyword arguments matching
+            required or recommended fields will be coerced to the appropriate
+            type, if necessary.
+
+        Examples
+        --------
+
+            gc = GardenClient()
+            pea_garden = gc.create_garden(
+                creators=["Mendel, Gregor"], title="Experiments on Plant Hybridization"
+            )
+            pea_garden.year = 1863
+        """
+        return Garden(**kwargs)
+
+    def register_garden(self, G: Garden):
```

Comment:
Capital letter for this parameter seems non-pythonic, right?

Reply:
agreed, also full-named parameters are preferred to abbreviations.

Reply:
indeed